### PR TITLE
Add header and some base components

### PR DESCRIPTION
**Pull request by VladSolodjankin**

### CODE REVIEW

**Comment by VladSolodjankin** [comment]
image

File: src/modules/Header/modules/MainMenu/MainMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      {children}
+    </Link>
+  );
+}
+
+export function MainMenu() {
+  return (
+    <AppBar className={cn('main-menu')}>
+      <MainMenuLink to="/">Home</MainMenuLink>
+      <MainMenuLink to="/appointments">Appointments</MainMenuLink>
```
Comment (EvgeniyKvasyuk) [inline src/modules/Header/modules/MainMenu/MainMenu.tsx:28]: all texts have to be in locale.ts only. It is strongly restricted to add text to components

Comment (VladSolodjankin) [reply src/modules/Header/modules/MainMenu/MainMenu.tsx]: done

**Pull request merged**